units/meson-add-wants.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -e
+
+unitdir="$1"
+target="$2"
+unit="$3"
+
+unitpath="${DESTDIR}${unitdir}/${unit}"
+dir="${DESTDIR}${unitdir}/${target}"
+
+if [[ "$target" == */ ]]; then
```

Comment:
btw, simple changing the shebang to ```#!/bin/bash``` does not work. Looks like meson.add_install_script() always uses /bin/sh:
```
Running custom install script '/bin/sh -e /home/michael/git/systemd/units/meson-add-wants.sh
```

@jpakkane Isn't that a bug in ```meson.add_install_script()```? What if the install script was implemented in, say, perl?